src/DevilDaggersInfo.Tools/ContentManager.cs

```diff
@@ -24,12 +24,13 @@ public static void Initialize()
 		if (!Directory.Exists(UserSettings.Model.DevilDaggersInstallationDirectory))
 			throw new InvalidGameInstallationException("Installation directory does not exist.");
 
-		// TODO: Use correct Linux file name for executable.
 #if WINDOWS
 		string ddExe = Path.Combine(UserSettings.Model.DevilDaggersInstallationDirectory, "dd.exe");
+#elif LINUX
+		string ddExe = Path.Combine(UserSettings.Model.DevilDaggersInstallationDirectory, "devildaggers");
```

Comment:
Thanks! Do you know if this differs between distros? I thought it was sometimes called `devildaggers.bin` but not sure.